package-scripts/datadog-agent/postinst

```diff
@@ -3,6 +3,8 @@
 CONFIG_DIR=/etc/dd-agent
 LOG_DIR=/var/log/datadog
 RUN_DIR=/opt/datadog-agent/run
+JMX_STATUS_PYTHON=/tmp/jmx_status_python.yaml
+JMX_STATUS=/tmp/jmx_status.yaml
```

Comment:
On Linux systems, temporary folders are supposed to get deleted at every `shutdown`, but restarting might be a problem for some of our users, wouldn't it ? 

We could actually call a small python script to retrieve the right folder just like we're doing it here (https://github.com/DataDog/dd-agent/blob/master/packaging/datadog-agent/source/install_agent.sh#L85)... it requires on working Python installation though... 

Another alternative would be to use the `/opt/datadog-agent-embedded/bin/python` if it's there... if not the Agent is installed so we don't care. What do you think @degemer @yannmh ?